### PR TITLE
6.4 shape

### PR DESCRIPTION
**Pull request by bkomorniczak**

### CODE REVIEW

File: kodilla-testing/build.gradle [hunk 1]
```diff
@@ -13,4 +13,5 @@ repositories {
 
 dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
+    testCompile 'com.github.stefanbirkner:system-rules:1.19.0'
```
Comment (piotr-grosicki) [inline kodilla-testing/build.gradle]: is it still needed?

File: kodilla-testing/src/main/java/com/kodilla/testing/shape/Circle.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package com.kodilla.testing.shape;
+
+public class Circle implements Shape{
+    private String name;
```
Comment (piotr-grosicki) [inline kodilla-testing/src/main/java/com/kodilla/testing/shape/Circle.java]: these fields could be final

File: kodilla-testing/src/main/java/com/kodilla/testing/shape/ShapeCollector.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package com.kodilla.testing.shape;
+
+import java.util.ArrayList;
+
+public class ShapeCollector {
+
+    ArrayList<Shape> shapes = new ArrayList <>();
```
Comment (piotr-grosicki) [inline kodilla-testing/src/main/java/com/kodilla/testing/shape/ShapeCollector.java]: should `private final`

File: kodilla-testing/src/main/java/com/kodilla/testing/shape/ShapeCollector.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+    public void addFigure(Shape shape){
+       shapes.add(shape);
+
+    }
+
+    public boolean removeFigure(Shape shape){
+        boolean result = false;
+        if (shapes.contains(shape)){
```
Comment (piotr-grosicki) [inline kodilla-testing/src/main/java/com/kodilla/testing/shape/ShapeCollector.java]: not needed just call `shapes.remove(shape)`. same result

File: kodilla-testing/src/main/java/com/kodilla/testing/shape/Square.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
+package com.kodilla.testing.shape;
+
+public class Square implements Shape{
+    private String name;
```
Comment (piotr-grosicki) [inline kodilla-testing/src/main/java/com/kodilla/testing/shape/Square.java]: can be final

File: kodilla-testing/src/main/java/com/kodilla/testing/shape/Triangle.java [hunk 6]
```diff
@@ -0,0 +1,28 @@
+package com.kodilla.testing.shape;
+
+public class Triangle implements Shape{
+    private String name;
```
Comment (piotr-grosicki) [inline kodilla-testing/src/main/java/com/kodilla/testing/shape/Triangle.java]: finals

File: kodilla-testing/build.gradle (hunk 1 above)
Comment (bkomorniczak) [reply kodilla-testing/build.gradle]: Nope, removing.

File: kodilla-testing/src/main/java/com/kodilla/testing/shape/Circle.java (hunk 2 above)
Comment (bkomorniczak) [reply kodilla-testing/src/main/java/com/kodilla/testing/shape/Circle.java]: Ok.

**Pull request merged**